platform/template/domain/src/main/kotlin/domain/EditorPayload.kt

```
@@ -0,0 +1,7 @@
+package domain
+
+data class EditorPayload(
+    val editor: String
+){
+
+}
```

detekt (1.23.8)

[warning] The class or object EditorPayload is empty.

(detekt.empty-blocks.EmptyClassBlock)